src/components/FilterButton/FilterButton.js

```diff
@@ -22,29 +22,37 @@ const propTypes = {
    */
   routeIdentifier: PropTypes.string.isRequired,
 
+  /**
+   * Button is active.
+   */
+  active: PropTypes.bool.isRequired,
+
+  /**
+   * Set active value.
+   */
+  setActive: PropTypes.func.isRequired,
```

Comment:
wierd name you already have an active property.  `onChange` seems more appropriate

Reply:
Indeed onChange is better :)